taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -77,16 +77,18 @@ def build_browsertime_task(config, tasks):
         abi = task["attributes"]["abi"]
         apk = task["attributes"]["apk"]
 
-        test_name = task.pop("test-name")
-
-        task["name"] = "{}-{}-{}".format(task["name"], build_type, abi)
-        task["description"] = "{}-{}".format(build_type, abi)
-
         for key in ("args", "treeherder.platform", "worker-type"):
             resolve_keyed_by(task, key, item_name=task["name"], **{"abi": abi})
 
         task["treeherder"] = inherit_treeherder_from_dep(task, signing)
 
+        test_name = task.pop("test-name")
+        platform = task["treeherder"]["platform"]
+
+        task_name = "{}-{}-{}-{}".format(platform, test_name, build_type, abi)
+        task["name"] = task_name
+        task["description"] = task_name
```

Comment:
Is there a reason why you're changing how the name and description of the task are built? This is quite different from what we currently have.

Here's an example of before and after:

Before:
```
  "metadata": {
    "name": "browsertime-tp6m-allrecipes-nightly-arm64-v8a",
    "owner": "cron@noreply.mozilla.org",
    "source": "https://github.com/mozilla-mobile/fenix/blob/85257dbb0af18efbbce40b4d92ab80b9982dc075/taskcluster/ci/browsertime",
    "description": "nightly-arm64-v8a ([Treeherder push](https://treeherder.mozilla.org/#/jobs?repo=fenix&revision=85257dbb0af18efbbce40b4d92ab80b9982dc075))"
  },
```

After:
```
  "metadata": {
    "name": "browsertime-android-hw-g5-7-0-arm7-api-16-shippable-qr/opt-allrecipes-nightly-armeabi-v7a",
    "owner": "davehunt@users.noreply.github.com",
    "source": "https://github.com/davehunt/fenix/blob/57af16cf8dcf19fd0711558f8f399880c0d5765a/taskcluster/ci/browsertime",
    "description": "android-hw-g5-7-0-arm7-api-16-shippable-qr/opt-allrecipes-nightly-armeabi-v7a ([Treeherder push](https://treeherder.mozilla.org/#/jobs?repo=fenix-pr&revision=57af16cf8dcf19fd0711558f8f399880c0d5765a))"
  },
```

Reply:
I believe we do at least need to include the platform as this indicates the WebRender status and without this we have duplicate names/descriptions. This also brings the task names/descriptions closer to what we have in m-c. For example allrecipes on moto G5 has a task name of 'test-android-hw-g5-7-0-arm7-api-16-shippable/opt-browsertime-tp6m-essential-geckoview-allrecipes-e10s'.